ShopifySharp/Services/InventoryLevel/InventoryLevelService.cs

```diff
@@ -33,5 +33,53 @@ public class InventoryLevelService : ShopifyService
 
             return await ExecuteRequestAsync<List<InventoryLevel>>(req, HttpMethod.Get, rootElement: "inventory_levels");
         }
+
+        /// <summary>
+        /// Deletes inventory for an item at specified location.  All items must keep inventory at at least one location.
+        /// </summary>
+        /// <param name="inventoryItemId">The ID of the inventory item.</param>
+        /// <param name="locationId">The ID of the location that the inventory level belongs to.</param>
+        public virtual async Task DeleteAsync(long inventoryItemId, long locationId)
+        {
+            await ExecuteRequestAsync(PrepareRequest($"inventory_levels.json?inventory_item_id={inventoryItemId}&location_id={locationId}"), HttpMethod.Delete);
+        }
+
+        /// <summary>
+        /// Updates the given <see cref="InventoryLevel"/>.
+        /// </summary>
+        /// <param name="updatedInventoryLevel">The updated <see cref="InventoryLevel"/></param>
+        /// <param name="disconnectIfNecessary">Whether inventory for any previously connected locations will be set to 0 and the locations disconnected. This property is ignored when no fulfillment service is involved.</param>
+        /// <returns>The updated <see cref="InventoryLevel"/></returns>
+        public virtual async Task<InventoryLevel> UpdateAsync(InventoryLevel updatedInventoryLevel, bool disconnectIfNecessary = false)
+        {
+            var req = PrepareRequest($"inventory_levels/set.json");
+            JsonContent content = new JsonContent(new
```

Comment:
Since three of these four properties come from the `Inventorylevel` object that's passed to the method, it would be better to convert the entire object to a dictionary and then add the `disconnect_if_necessary` prop to that dictionary:

```cs
var body = updatedInventoryLevel.ToDictionary();
body.Add("disconnect_if_necessary", disconnectIfNecessary);

var content = new JsonContent(body);
```